qp_pacbio/tests/test_pacbio.py

```diff
@@ -387,5 +388,108 @@ def test_syndna(self):
         self.assertEqual(obs_finish, exp_merge)
 
 
+class PacAdapterRmTests(PacBioTests):
+    def test_pacbio_adapter_removal(self):
+        params = {
+            "artifact": int(self._insert_data()),
+            "adapter": "AAGCAGTGGTATCAACGCAGAGTACT",
+            "css": False,
```

Comment:
These look copied, why not import them? Why do the parameters exist both typed and as str?